fem/fe.hpp

```diff
@@ -1207,18 +1296,21 @@ class CrouzeixRaviartQuadFiniteElement : public NodalFiniteElement
 class P0SegmentFiniteElement : public NodalFiniteElement
 {
 public:
+   /// Construct the FiniteElement with dummy order @a Ord
    P0SegmentFiniteElement(int Ord = 0);
    virtual void CalcShape(const IntegrationPoint &ip, Vector &shape) const;
    virtual void CalcDShape(const IntegrationPoint &ip,
                            DenseMatrix &dshape) const;
 };
 
+/** @brief A 2D 1st Raviart-Thomas vector element on a triangle */
```

Comment:
Does this mean 1st order?  If so, then perhaps the word "order" should be added to this and the next few class descriptions.